packages/web/components/Post/components/InputComment.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+
+export const InputComment: React.FC = React.memo(() => {
+  return (
+    <div
```

Comment:
`input` musi być wewnątrz `form` i powinien mieć `label`

Reply:
Zamień `div` na `form` i dodaj `label` do inputa :)

Reply:
Dodałem `<form>` ale labela nie chcemy wyświetlać (instagram nie wyświetla w tym miejscu nic).

Reply:
Label musi być ze względu na czytniki ekranowe. Jak nie chcemy by się label pokaywał, to można go ukryć css-em.